packages/trader/src/App/Containers/Redirect/redirect.jsx

```diff
@@ -10,11 +10,14 @@ const Redirect = ({
     setVerificationCode,
     toggleAccountSignupModal,
     toggleCashierModal,
+    toggleResetPasswordModal,
 }) => {
     const url_params = new URLSearchParams(window.location.search);
 
     setVerificationCode(url_params.get('code'));
-
+    
+    // eslint-disable-next-line no-console
+    console.log(url_params.get('action'));
```

Comment:
do we need this log?

Reply:
nice catch, yep, we dont need this log, will remove 👍